# DeleteRobot.srv
uint32 robot_id
---
bool success
string message

---